GeometryService/src/STMMaker.cc

```diff
@@ -349,10 +349,45 @@ namespace mu2e {
                                _STM_SSCoffset_Spot,
                                _STM_SSCleak,
                                _STM_SSCFrontToWall,
+                               _STM_SSCZGap,
+                               _STM_SSCZGapBack,
                                _STM_SSCOffsetInMu2e,
                                _STM_SSCRotation,
                                _STM_SSCMaterial));
 
+   ////////////////////////////////////////////////////////////////
+   //Spot-Size Collimator Support
+
+    const CLHEP::HepRotation _SSCSupportRotation     = CLHEP::HepRotation::IDENTITY;
+    const CLHEP::Hep3Vector  _SSCSupportOffsetInMu2e = _STMShieldingRef + CLHEP::Hep3Vector(0, 0, _STM_SSCZGap/2);
+
```

Comment:
A simple data member could be used instead of a unique_ptr in this class, I would consider this possibility in future development work